### DISCUSSION TITLE
V23(Beta)刚开机时数据盘文件突然增加

### DISCUSSION THREAD

**Discussion by Aninana** [linuxdeepin/developer-center#4454] [General | 常规讨论]
原帖：https://bbs.deepin.org/post/257055
刚开机时，选中数据盘，然后就看到不断的新增文件数量，一分钟不到，文件数量就增加了16301个。而且这个盘还没有存入其他文件，但已经用了11G多了。
这是bug还是系统设置需要？
![1](https://github.com/linuxdeepin/developer-center/assets/130546027/84ff76e5-d2d8-41da-a512-ef0650613fcc)

**Comment by BLumia** [comment]
cc @babyfengfjx
